client/src/game/board/cards/UnitCard.tsx

```diff
@@ -29,9 +29,8 @@ export class UnitCard extends React.Component<UnitCardT & {displayOverlay?: bool
             })
         }
 
-        let statsText = this.props.move + " / " ;
-        statsText += this.props.attack  + " / ";
-        statsText += this.props.health;
+        let statsText = <div className={css.statsOverlayContainer}><p className={css.statsOverlayText}>{this.props.move}</p>
+        <p className={css.statsOverlayText}>{this.props.attack}/{initialHealth}</p></div>
```

Comment:
The formatting here is inconsistent. We should use ctrl+alt+L to fix the formatting